dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml

```diff
@@ -40,7 +40,7 @@ dim_config:
       params:
         columns:
           - country
-        regex: ^[A-Z]{2}$  # example: GB
+        regex: ^[A-Z]{2}|\?\?$  # example: GB or ??
```

Comment:
?? is a valid value, means undefined and it's used to join with the country lookup.

Reply:
yes, I was just wondering if maybe we do that to avoid pipelines breaking but this value was maybe "undesirable".